src/cascadia/TerminalApp/ActionArgs.cpp

```diff
@@ -167,6 +168,16 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    winrt::hstring SendInputArgs::GenerateName() const
+    {
+        // The string will be similar to the following:
+        // * "Send Input: ...input..."
+
+        return winrt::hstring{
+            fmt::format(L"{}: {}", RS_(L"SendInputCommandKey"), _Input)

```

Comment:
Typically, it's best practice to have just two entirely different resources in Resources.resw for something like this.

In this case though, I don't think we need the version without the argument at all, we could probably get away with a single resource for the formatted version:

```xml

  <data name="SendInputCommandKey" xml:space="preserve">
    <value>Send Input: "{0}"</value>
    <comment>{0} will be replaced with a string of input as defined by the user.</comment>
  </data>
```

Reply:
I'm suddenly worried about dropping the user's input right into a TextBlock. Should we transform the control characters <0x20 to their "[control pictures](https://en.wikipedia.org/wiki/Control_Pictures)" equivalents? It's as easy as adding 0x2400 to them!

Reply:
This is what we do in the debug tap ... which I can't get a screenshot of because we broke it o_O

Reply:
Or perhaps my right alt key no longer works. This would come as no big surprise.

Reply:
>  Should we transform the control characters <0x20 to their "control pictures" equivalents? It's as easy as adding 0x2400 to them!

I'm hugely in favor of this actually. Raw control characters might _definitely_ break the text block